HSTracker/Utility/BoardDamage/BoardHero.swift

```diff
@@ -12,24 +12,36 @@ class BoardHero: IBoardEntity {
     private(set) var _baseAttack = 0
     private(set) var _hero: BoardCard
     private(set) var _weapon: BoardCard?
-    
-    var name: String { return _hero.name }
-    var hasWeapon: Bool { return _weapon != nil }
-    
+
+    var name: String {
```

Comment:
I think I will revert this, I just pressed format code in appcode but actually the original version is quicker to read